website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md

```diff
@@ -77,6 +77,17 @@ As example:
 dbt build --select result:error+ result:fail+
 ```
 
+As an extension of the Slim CI feature, dbt Cloud can rerun and retest only the things that are fresher compared to a previous run.
```

Comment:
- Let's `VersionBlock` this section
- I'm hesitant to include this here until we've confirmed for sure that this works in Cloud—and even then, while it's experimental / until it's a bit better tested

Reply:
added a version block and left in there for now.